discord/message.go

```diff
@@ -181,20 +181,28 @@ type Sticker struct {
 	// ID is the ID of the sticker.
 	ID StickerID `json:"id"`
 	// PackID is the ID of the pack the sticker is from.
-	PackID StickerPackID `json:"pack_id"`
+	PackID StickerPackID `json:"pack_id,omitempty"`
 	// Name is the name of the sticker.
 	Name string `json:"name"`
 	// Description is the description of the sticker.
 	Description string `json:"description"`
 	// Tags is a comma-delimited list of tags for the sticker. To get the list
 	// as a slice, use TagList.
 	Tags string `json:"-"`
-	// Asset is the sticker's assert hash.
-	Asset Hash `json:"asset"`
+	// The type of sticker
+	Type StickerType `json:"type"`
```

Comment:
```suggestion
	// Type is the type of sticker.
	Type StickerType `json:"type"`
```
Go comments always start with the name of the entity.